koku/masu/processor/ocp/ocp_cost_model_cost_updater.py

```diff
@@ -514,24 +513,6 @@ def distribute_costs_and_update_ui_summary(self, start_date, end_date):
                 report_period.derived_cost_datetime = timezone.now()
                 report_period.save()
 
-    def _update_vm_count_tag_based_cost(self, start_date, end_date):
```

Comment:
This method could have remained a wrapper around `populate_vm_count_tag_based_costs` just like `_update_tag_usage_costs` and `_update_tag_usage_default_costs` and `_delete_tag_usage_costs` are wrappers.